internal/contour/listener.go

```diff
@@ -261,8 +274,7 @@ func (v *listenerVisitor) visit(vertex dag.Vertex) {
 
 		// attach certificate data to this listener if provided.
 		if vh.Secret != nil {
-			data := vh.Secret.Data()
-			fc.TlsContext = envoy.DownstreamTLSContext(data[v1.TLSCertKey], data[v1.TLSPrivateKeyKey], vh.MinProtoVersion, alpnProtos...)
+			fc.TlsContext = envoy.DownstreamTLSContext(envoy.Secretname(vh.Secret), v.xdsClusterName(), vh.MinProtoVersion, alpnProtos...)
```

Comment:
please don't make v.xdsClusterName() configurable, please delete it

Reply:
I'm sorry. I assumed the comment about `ListenerVistorConfig` [defaults](https://github.com/heptio/contour/pull/1049#discussion_r281057617) meant, remove the flags that control them, and convert to defaults like the others. Thank you for clean it up for me.